src/PropertyAppTests/JsonReadWriteTests/IssueModels.json

```diff
@@ -43,4 +43,4 @@
     },
     "capturedDateAndTime": "2023-10-25T17:43:20"
   }
-]
+]
```

Comment:
This one will _constantly_ be getting reverted back to no blank line at end of the file because of the json handler modifying the file